app/controllers/carriers_controller.rb

```diff
@@ -0,0 +1,43 @@
+require 'timeout'
```

Comment:
You require timeout here, but I think the code for this got moved to the model / was commented out; remember to remove code that is no longer being used by a file.